src/node/config.ts

```diff
@@ -6,6 +6,7 @@ import { resolveAliases, APP_PATH, DEFAULT_THEME_PATH } from './alias'
 import { SiteData, HeadConfig, LocaleConfig } from '../../types/shared'
 import { MarkdownOptions } from './markdown/markdown'
 import { AliasOptions } from 'vite'
+import { Options } from '@vitejs/plugin-vue'
```

Comment:
We can alias this to `VuePluginOptions` to provide the context

Reply:
Sure!